tests/laser/transaction/symbolic_test.py

```diff
@@ -67,3 +70,21 @@ def test_execute_contract_creation(mocked_setup: MagicMock):
     # laser_evm.exec.assert_called_once()
     assert laser_evm.exec.call_count == 1
     assert len(laser_evm.open_states) == 0
+
+
+def test_calldata_constraints_in_transaction():
+    # Arrange
+    laser_evm = LaserEVM({})
+    world_state = WorldState()
+
+    transaction = MessageCallTransaction(
+        world_state, Account("ca11ee"), Account("ca114")
```

Comment:
Can you put in a Mocked calldata, where the function constraints returns a defined set of constraints. 
At assert we can then check if those specific constraints get added.
WDYT?

Reply:
Alright, I've updated the test case.